tests/manage/rgw/test_host_node_failure.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.resources.pod import (
+    get_rgw_pods, get_pod_node, get_noobaa_pods
+)
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework.testlib import (
+    ManageTest, tier4
+)
+from tests.sanity_helpers import Sanity
+from tests.helpers import wait_for_resource_state
+from ocs_ci.ocs.node import get_node_objs
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@pytest.mark.polarion_id("OCS-2374")
+class TestRGWHostNodeFailure(ManageTest):
```

Comment:
We are failing the node where both noobaa-db-0 and rgw pod are hosting, please update the log according to it. 
Also the test case name

Reply:
done